### DISCUSSION TITLE
FileStorage - S3/Amazon - ServerSideEncryption

### DISCUSSION THREAD

**Discussion by onurozkan** [directus/directus#15203] [Feature Requests]
Hey,

We are using serverside encryption on s3 file storage as default. I checked directus options but I couldn't find any connected implementation so far. 

I believe we only need to pass `ServerSideEncryption` option to put (and maybe some other methods too), then aws-sdk should cover the rest.

https://github.com/directus/directus/blob/main/packages/drive-s3/src/AmazonWebServicesS3Storage.ts#L266

Is there any reason why its not implemented? If not, I can try to raise a PR to cover sse issue.

Thanks.

**Comment by rijkvanzanten** [comment]
> Is there any reason why its not implemented? If not, I can try to raise a PR to cover sse issue.

Not really! Only reason it doesn't exist now is that the request never came up before. Feel free to open a PR for that tweak 🙂

**Comment by rijkvanzanten** [comment]
This was since implemented
